src/ui-components/OnchainCreationLabel.tsx

```diff
@@ -41,14 +40,7 @@ const OnchainCreationLabel = ({ address, topic, username, truncateUsername }:Pro
 						: null
 				}
 			</div>
-			{
-				topic?
-					<div className='flex items-center'>
-						<div className='mr-1.5 ml-auto hidden min-[340px]:flex'>from</div>
-						<TopicTag topic={topic} />
```

Comment:
TopicTag why we remove

Reply:
TopicTag added in a different place to align with designs